zcash_primitives/src/transaction/components/sapling/builder.rs

```diff
@@ -336,22 +429,18 @@ impl<P: consensus::Parameters> SaplingBuilder<P> {
         Ok(())
     }
 
-    pub fn build<Pr: TxProver, R: RngCore>(
+    pub fn build<SP: SpendProver, OP: OutputProver, R: RngCore>(
```

Comment:
I don't see how these type parameters are being used; they don't show up either in the inputs or outputs to this function?